app/poros/piece.rb

```diff
@@ -0,0 +1,9 @@
+class Piece
```

Comment:
as I mentioned in slack, I made a model for Piece. Could you pull master into your branch, and redo your checks.